apps/angular/bug-cd/src/app/menus.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'menus',
+  standalone: true,
+})
+export class MenusPipe implements PipeTransform {
+  transform(prop: string) {
+    return this.getMenu(prop);
+  }
+
+  getMenu(prop: string) {
```

Comment:
That's a little bit over-engineered. The `transform` function can return his string array too, no need to create a extra function for it, but that's just my opinion. But I'd define the return type of the `transform` function, which should be `string[]`.

Reply:
I got your point